src/components/flowchart/flowchart.js

```diff
@@ -220,22 +220,28 @@ export class FlowChart extends Component {
 
     if (changed('edges', 'nodes', 'layers', 'chartSize', 'clickedNode')) {
       // Don't zoom out when the metadata or code panels are opened or closed
```

Comment:
is this PR to not refocus flowchart due to any change or just the node click ? 

if it is not to refocus in anyway may be a check here like below should work without any other code change I guess ?

```javascript
if (!this.props.nodeRefocus) {
return;
}
```

Reply:
@ravi-kumar-pilla I started with this approach, but it breaks the flowchart on the first load because `this.resetView` does not execute, as we are returning early. Here, `this.resetView` is responsible for centering and refocusing the flowchart.